wallet/src/Pos/Util/Mnemonic.hs

```diff
@@ -0,0 +1,683 @@
+-- | Module providing restoring from backup phrase functionality
+{-# LANGUAGE DataKinds      #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE TypeFamilies   #-}
+{-# OPTIONS_GHC -Wno-redundant-constraints #-}
```

Comment:
Some of the type-family `SupportedWords` are judged _redundant_ while they're there to enforce we use a supported `Nat`